File: challenges/2-idade-em-dias/go/KMarculo/desafio2.go
package main

import (
	"fmt"
)

func main() {
	var idadeEmDias int

	fmt.Println("Informe a idade em dias.")
	fmt.Scan(&idadeEmDias)

	anos := idadeEmDias / 365
	if anos < 0 {
		anos = 0
	}
	meses := (idadeEmDias - (anos * 365)) / 30
	if meses < 0 {
		meses = 0
	}
	dias := idadeEmDias - (meses * 30) - (anos * 365)
	if dias < 0 {
		dias = 0
	}

	fmt.Printf("%d ano(os)\n%d mes(es)\n%d dia(as) \n", anos, meses, dias)

}

File: challenges/2-idade-em-dias/ruby/laisacsts/desafio-2.rb
print "Digite a idade em dias (ex: 400): "
idade_em_dias = gets.chomp.to_i

ref_ano = 365
ref_mes_e_dia = 30

indica_ano = idade_em_dias / ref_ano
indica_mes = (idade_em_dias % ref_ano) / ref_mes_e_dia
indica_dia = (idade_em_dias % ref_ano) % ref_mes_e_dia

puts "#{indica_ano} ano(s)"
puts "#{indica_mes} mês(es)"
puts "#{indica_dia} dia(s)"
